<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>directory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="directoryLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse..</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>output filename</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputFilenameLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="splitButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>split</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QComboBox" name="concatTypeComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>wav</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mp3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>m4a</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>flac</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mp4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>wma</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>aac</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
